### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by aleixalcacer**

### CODE REVIEW

File: docs/sphinx/conf.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 subprocess.call('cd ../doxygen && doxygen Doxyfile && cd ../sphinx', shell=True)
 
 project = 'Caterva'
-copyright = '2019-2020, Blosc Developers'
+copyright = '2020, Blosc Developers'
```
Comment (FrancescAlted) [inline docs/sphinx/conf.py]: You need to keep expressing that copyright starts in 2019.  So 2019-2020 is correct here.

File: docs/sphinx/getting-started/description.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Caterva is a container for multidimensional data that is specially designed to
+read, in an incredibly efficient way, slices of it. To achieve this, a new
+chunking-based data layout has been created.
+
+.. image:: overview.png
+   :width: 40%
+   :align: center
+
+Like other libraries like zarr, hdf5 or TileDB, Caterva stores the data into
```
Comment (FrancescAlted) [inline docs/sphinx/getting-started/description.rst]: HDF5 is normally written in upper case.

File: docs/sphinx/getting-started/description.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+multidimensional chunks (yellow cubes). These chunks can then be read
+individually, improving performance when reading slices of the dataset. But
+also, Caterva introduces a new level of chunking. Within each chunk, the data is
+re-partitioned into smaller multidimensional sets called blocks (green cubes).
+In this way, Caterva can read blocks individually (and also in parallel) instead
+of chunks.
+
+These partition levels, that emulate the memory hierarchy of modern computers,
```
Comment (FrancescAlted) [inline docs/sphinx/getting-started/description.rst]: I am not sure that partition levels emulate cache hierarchy of computers.  As this is complex point, I suggest to just say that allow to access data efficiently with a larger set of data access patterns.

File: docs/sphinx/getting-started/description.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+cannot be persisted. It is critical to find a way to store and retrieve data
+efficiently. Also, it is important to adopt open formats for reducing the
+maintenance burden and facilitate its adoption more quickly. Blosc2 brings such
+an efficient and open format for `persistency <https://github
+.com/Blosc/c-blosc2/blob/master/README_FRAME_FORMAT.rst>`__.
+
+An aditional feature that introduces Blosc2 is the concept of metalayers. They
+are small metadata for informing about the kind of data that is stored on a
+Blosc2 container. They are handy for defining layers with different specs: data
+types, geo-spatial...
```
Comment (FrancescAlted) [inline docs/sphinx/getting-started/description.rst:45]: I suppose this is also repeated in README.md.  I am wondering whether it is feasible to centralize this description somewhere and include from here.  Not extremely important, but planning for a single source of prose is always a good thing.

**Review by FrancescAlted — CHANGES_REQUESTED** [review]

File: docs/sphinx/conf.py (hunk 1 above)
Comment (aleixalcacer) [reply docs/sphinx/conf.py]: Fixed

File: docs/sphinx/getting-started/description.rst (hunk 3 above)
Comment (aleixalcacer) [reply docs/sphinx/getting-started/description.rst]: Fixed

File: docs/sphinx/getting-started/description.rst (hunk 2 above)
Comment (aleixalcacer) [reply docs/sphinx/getting-started/description.rst]: Fixed

File: docs/sphinx/getting-started/description.rst (hunk 4 above)
Comment (aleixalcacer) [reply docs/sphinx/getting-started/description.rst:45]: In README.md there is only a brief description of Caterva and a link to the documentation. So there should be no duplicate text. 

Make sense?

Comment (FrancescAlted) [reply docs/sphinx/getting-started/description.rst:45]: Yes, it does.

**Review by FrancescAlted — APPROVED** [review]

**Pull request merged**